.changeset/thick-feet-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/cli-hydrogen': patch
+---
+
+bump @shopify/oxygen-cli from 2.3.2 to 2.6.1
```

Comment:
I think this might be confusing for users since they are not aware of any "oxygen cli". Instead of bumping a dependency, can you mention what is this PR actually fixing? Let's write changelogs for the users, we can always check commits/PRs if we devs need references 👍